src/App.js

```diff
@@ -746,19 +767,36 @@ const App = () => {
                 statusHandler={resetLoadStatus}
               />
             </div>
-            <div className="row mt-3">
-              <div
+            <div
+              style={{
+                display: "flex",
+                flexDirection: "row",
+                justifyContent: "center",
+                alignItems: "center",
+                marginTop: 10,
+                height: 40,
+                width: "100%",
+              }}
+            >
+              <button
                 style={{
-                  display: "flex",
-                  flexDirection: "column",
-                  justifyContent: "center",
-                  alignItems: "center",
+                  border: "none",
+                  borderRadius: 10,
+                  height: "90%",
+                  width: 150,
+                }}
+                onClick={() => {
+                  document.getElementById("searchBox").style.display =
+                    showSearchBox ? "none" : "block";
+                  setShowSearchBox((x) => !x);
                 }}
               >
-                More Filters
-              </div>
+                {showSearchBox ? "Hide Filters" : "More filters"}
```

Comment:
![image](https://github.com/eluv-io/elv-clip-search/assets/52431045/cf5b13d8-edfc-4c66-b7e4-2cdca82bcdfc)
- `More filters` and `Add keywords` appear at the same time when login account. 

- The app should hide `Add keywords` filter by default.